OSPSuite.Presentation.Importer/Views/ImporterTiledView.cs

```diff
@@ -0,0 +1,47 @@
+﻿using OSPSuite.Presentation.Importer.Presenters;
+using OSPSuite.UI.Extensions;
+using System;
+
+namespace OSPSuite.Presentation.Importer.Views
+{
+   public partial class ImporterTiledView
+   {
+      private IImporterTiledPresenter _presenter;
+
+      public ImporterTiledView()
+      {
+         InitializeComponent();
+         navigationTileBar.ItemClick += onTileBarClicked;
+      }
+
+      private void onTileBarClicked(object sender, EventArgs e)
+      {
+         var eventArgs = e as DevExpress.XtraEditors.TileItemEventArgs;
+         if (eventArgs.Item.Name == "dataMappingTile")
```

Comment:
what are those magic strings here?Define cosntants

Reply:
I think there are lots of strings used on the ImporterTiledView. We need to used captions and then the problem will be solved by using those captions.